varats/varats/projects/c_projects/grep.py

```diff
@@ -0,0 +1,75 @@
+"""Project file for the GNU grep."""
+import typing as tp
+from pathlib import Path
+
+import benchbuild as bb
+from benchbuild.utils.cmd import make
+from benchbuild.utils.settings import get_number_of_jobs
+from plumbum import local
+
+from varats.paper_mgmt.paper_config import project_filter_generator
+from varats.project.project_util import (
+    wrap_paths_to_binaries,
+    ProjectBinaryWrapper,
+    BinaryType,
+)
+from varats.provider.cve.cve_provider import CVEProviderHook
+from varats.utils.settings import bb_cfg
+
+
+class Grep(bb.Project, CVEProviderHook):  # type: ignore
+    """GNU Grep / UNIX command-line tools (fetched by Git)"""
+
+    NAME = 'grep'
+    GROUP = 'c_projects'
+    DOMAIN = 'utils'
+
+    SOURCE = [
+        bb.source.Git(
+            remote="https://git.savannah.gnu.org/git/grep.git",
+            local="grep",
+            refspec="HEAD",
+            limit=None,
+            shallow=False,
+            version_filter=project_filter_generator("grep")
+        ),
+        bb.source.GitSubmodule(
+            remote="https://git.savannah.gnu.org/git/gnulib.git",
+            local="grep/gnulib",
+            refspec="HEAD",
+            limit=None,
+            shallow=False,
+            version_filter=project_filter_generator("grep")
+        )
+    ]
+
+    @property
+    def binaries(self) -> tp.List[ProjectBinaryWrapper]:
+        """Return a list of binaries generated by the project."""
+
+        grep_source = local.path(self.source_of_primary)
+        return wrap_paths_to_binaries([
+            (grep_source / "src/grep", BinaryType.executable),
+        ])
```

Comment:
```suggestion
        return wrap_paths_to_binaries([
            ("src/grep", BinaryType.executable),
        ])
```
Paths for binaries are given relative to the project root.